Add URL to WebSocket API documentation (#1202)
home-assistant/developers.home-assistant: docs/api/websocket.md

title: "WebSocket API"
---

Home Assistant hosts a WebSocket API at `/api/websocket`. This API can be used to stream information from a Home Assistant instance to any client that implements WebSockets. We maintain a [JavaScript library](https://github.com/home-assistant/home-assistant-js-websocket) which we use in our frontend.

> The WebSocket API uses the same port as the web frontend. (default port is port 8123).
> - `http://IP_ADDRESS:8123/` is an interface to control Home Assistant.
> - `ws://IP_ADDRESS:8123/api/websocket` is the WebSocket endpoint

Reviewer: ```suggestion
```

> 
> The WebSocket API uses the same port as the web frontend. (default port is port 8123).
> - `http://IP_ADDRESS:8123/` is an interface to control Home Assistant.
> - `ws://IP_ADDRESS:8123/api/websocket` is the WebSocket endpoint

Reviewer: ```suggestion
```

> Home Assistant contains a WebSocket API. This API can be used to stream information from a Home Assistant instance to any client that implements WebSockets. We maintain a [JavaScript library](https://github.com/home-assistant/home-assistant-js-websocket) which we use in our frontend.

Reviewer 2: Just put it in the first sentence.

```suggestion
Home Assistant hosts a WebSocket API at `/api/websocket`. This API can be used to stream information from a Home Assistant instance to any client that implements WebSockets. We maintain a [JavaScript library](https://github.com/home-assistant/home-assistant-js-websocket) which we use in our frontend.
```

> The WebSocket API uses the same port as the web frontend. (default port is port 8123).
> - `http://IP_ADDRESS:8123/` is an interface to control Home Assistant.
> - `ws://IP_ADDRESS:8123/api/websocket` is the WebSocket endpoint
> 

Reviewer 2: ```suggestion
```

Author: I did this mostly to be consistent with the REST API page. It's admittedly a bit verbose, but IMHO it's really helpful to beginners to provide the full URL (including the port number). Like I said, I'm new to websockets and Home Assistant. I didn't know if for example, the application was listening on a separate port for websocket connections.

Reviewer: I agree with the suggestion. Please note, this is the developer docs, which is not aimed at beginners.

../Frenck
